simexpal/base.py

```diff
@@ -750,6 +750,15 @@ def is_compiled(self):
 	def is_installed(self):
 		return os.access(os.path.join(self.prefix_dir, 'installed.simexpal'), os.F_OK)
 
+	def purge(self):
+		if not self.revision.is_dev_build:
+			util.try_rmtree(self.clone_dir)
+		else:
+			util.try_rmtree(self.source_dir)
```

Comment:
This deletes the source directory for dev builds. I think that is quite unexpected.

Reply:
The source dir for dev builds will only be deleted if an extra arg `--delete_source` is used in the purge command